server-spi/src/main/java/org/keycloak/models/RealmModel.java

```diff
@@ -307,7 +307,7 @@
     List<IdentityProviderModel> getIdentityProviders();
     IdentityProviderModel getIdentityProviderByAlias(String alias);
     void addIdentityProvider(IdentityProviderModel identityProvider);
-    void removeIdentityProviderByAlias(String alias);
+    boolean removeIdentityProviderByAlias(String alias);
```

Comment:
This change is not needed, the `removeIdentityProviderByAlias` either succeeds or throws an exception.

Reply:
Return to void